src/client/components/configuration/style.css

```diff
@@ -0,0 +1,48 @@
+.configuration {
+	flex-grow: 1;
+	flex-shrink: 1;
+	display: flex;
+	flex-direction: column;
+    font-weight: normal;

```

Comment:
Could you reformat all these CSS files to use 2 space indentation? 🙂 I think its my fault for checking in some non-2-space files.